### PR TITLE
Fixing broken links for PowerMax

### PR DESCRIPTION
**Pull request by boyamurthy**
Fixing broken links for PowerMax

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
|https://github.com/dell/csm/issues/1072 |

# Checklist:

- [X] Have you run a grammar and spell checks against your submission?
- [ ] Have you tested the changes locally?
- [ ] Have you tested whether the hyperlinks are working properly?
- [ ] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

**Review by tssushma — APPROVED** [review]
lgtm

**Review by shanmydell — DISMISSED** [review]

File: content/docs/deployment/csmoperator/drivers/powermax.md [hunk 1]
```diff
@@ -180,6 +180,21 @@ Set up the environment as follows:
 - Add initiators from all ESX/ESXis to a host(initiator group)/host group(cascaded initiator group) where the cluster is hosted.
 - Create a secret which contains vCenter privileges. Follow the steps [here](#support-for-auto-rdm-for-vsphere-over-fc) to create the same.
 
+### CSI PowerMax ReverseProxy
+
+CSI PowerMax ReverseProxy is component that will be installed along with the CSI PowerMax driver. For more details on this feature see the related [documentation](https://dell.github.io/csm-docs/docs/csidriver/features/powermax/#csi-powermax-reverse-proxy).
```
Comment (gallacher) [inline content/docs/deployment/csmoperator/drivers/powermax.md:185]: Replace with "CSI PowerMax ReverseProxy is a component that will be installed with the CSI PowerMax driver. For more details on this feature, see the related [documentation](https://dell.github.io/csm-docs/docs/csidriver/features/powermax/#csi-powermax-reverse-proxy)."

File: content/docs/deployment/csmoperator/drivers/powermax.md [hunk 2]
```diff
@@ -180,6 +180,21 @@ Set up the environment as follows:
...
+
+#### Pre-requisites
+Create a TLS secret that holds an SSL certificate and a private key which is required by the reverse proxy server.
```
Comment (gallacher) [inline content/docs/deployment/csmoperator/drivers/powermax.md:188]: Replace with "Create a TLS secret that holds an SSL certificate and a private key. This is required by the reverse proxy server."

**Review by gallacher — CHANGES_REQUESTED** [review]
Address review comments

File: content/docs/deployment/csmoperator/drivers/powermax.md (hunk 1 above)
Comment (mareksuski-dell) [reply content/docs/deployment/csmoperator/drivers/powermax.md]: I have included your comment

**Review by mareksuski-dell — APPROVED** [review]
LGTM

**Review by EmilyKatdell — DISMISSED** [review]

**Review by mareksuski-dell — APPROVED** [review]
LGTM

**Pull request merged**